Vagrantfile

```diff
@@ -5,43 +5,38 @@ require 'yaml'
 require "#{File.dirname(__FILE__)}/lib/forklift"
 
 VAGRANTFILE_API_VERSION = '2'
-SUPPORT_SSH_INSERT_KEY = Gem.loaded_specs['vagrant'].version >= Gem::Version.create('1.7')
-SUPPORT_NAMED_PROVISIONERS = Gem.loaded_specs['vagrant'].version >= Gem::Version.create('1.7')
-SUPPORT_BOX_CHECK_UPDATE = Gem.loaded_specs['vagrant'].version >= Gem::Version.create('1.5')
```

Comment:
We have a requirement for local usage, but continue to maintain compability due to the CI system as far as I know. But we need to double check before we remove any of this.